app/src/Controller.php

```diff
@@ -51,12 +51,122 @@ public function index(Request $request, Response $response, array $args)
         in_array($layout, ['grid', 'list']) or $layout = 'grid';
 
         $template = $this->app['twig']->loadTemplate(sprintf('layout/%s.twig', $layout));
-        $response->setContent($template->render(['instances' => $this->app['manager']->getAll()]));
+        $response->setContent($template->render(['instances' => $this->app['manager']->getAll(),'baseUrl'=> $this->app->getConfig('dir','/'), 'globalActions' => $this->app->getConfig('globalActions',false)]));
 
         return $response;
     }
 
     /**
+     * Starts all processes
+     *
+     * @param Request  $request
+     * @param Response $response
+     * @param array    $args
+     *
+     * @return Response
+     */
+    public function startAllServers(Request $request, Response $response, array $args)
+    {
+        $instances = $this->app['manager']->getAll();
+        foreach ($instances as $instance) {
+             $instance->startAllProcesses($this->app->getConfig('waitForSupervisor', false));
+        }
+
+        return new RedirectResponse($this->app->getConfig('dir','/'));
+    }
+
+    /**
+     * Restarts all processes
+     *
+     * @param Request  $request
+     * @param Response $response
+     * @param array    $args
+     *
+     * @return Response
+     */
+    public function restartAllServers(Request $request, Response $response, array $args)
+    {
+        $instances = $this->app['manager']->getAll();
+        foreach ($instances as $instance) {
```

Comment:
Can you please add a few newlines to increase readability? Like before and after loops?